src/db/models.ts

```diff
@@ -178,8 +178,54 @@ export const CIP25MetadataAsset = CIP25Metadata.belongsTo(Asset, {
 });
 export const AssetCIP25Metadata = Asset.hasOne(CIP25Metadata);
 
-export const CIP25MetadataBlock = CIP25Metadata.belongsTo(Asset, {
+export const CIP25MetadataBlock = CIP25Metadata.belongsTo(Block, {
   onDelete: "CASCADE",
   onUpdate: "CASCADE",
 });
 Block.hasOne(CIP25Metadata);
+
+/**
+ * CIP 68 is annoying as there is no way to differentiate between different
+ * metadata
+ */
+export class CIP68Metadata extends Model<
+  InferAttributes<CIP68Metadata>,
+  InferCreationAttributes<CIP68Metadata>
+> {
+  declare subject: string;
+  declare name: string;
+  declare description?: string;
+  declare otherProperties?: string;
```

Comment:
a lot of stuff is pushed here. A different DB might be better served for this (e.g. postgre with jsonb)